app/components/overview-info-form/component.js

```diff
@@ -1,11 +1,13 @@
 import Ember from 'ember';
+import {translations} from '../../utils/translationStrings';
 import {validator, buildValidations} from 'ember-cp-validations';
 
 
 var range = function(start, stop) {
     var options = [];
     for (var i=start; i <= stop; i++) {
-        options.push(i);
+      var key = 'number' + i;
```

Comment:
Sorry, I should have been more clear. The first line of the function body (L7) is indented 4 spaces, whereas you only use 2 inside of the loop. I'm fine with either 4 or 2 spaces, but we should try to stick with one.